traits/src/float.rs

```diff
@@ -337,27 +337,25 @@ pub trait Float
     ///
     /// assert!(f.is_sign_positive());
     /// assert!(!g.is_sign_positive());
-    /// // Requires both tests to determine if is `NaN`
-    /// assert!(!nan.is_sign_positive() && !nan.is_sign_negative());
+    /// assert!(!nan.is_sign_negative());
```

Comment:
Hmm, we should be testing `fn is_sign_positive()` here, but I realize older rust won't return true.  Perhaps this one should check `!neg_nan.is_sign_positive()`.